bin/bump-version

```diff
@@ -49,7 +49,7 @@ else
     done
 fi
 
-rm -f src/{clusterd,environmentd,persist-client,stash-debug}/Cargo.toml.bak LICENSE.bak
+rm -f src/{clusterd,environmentd,persist-client,testdrive,stash-debug}/Cargo.toml.bak LICENSE.bak
```

Comment:
If I was better at bash, I would save these binaries in a variable so we could be sure that this always matches line 32. Unfortunately I spent about 10 minutes trying to do that and then gave up.